iina/AutoFileMatcher.swift

```diff
@@ -45,9 +46,11 @@ class AutoFileMatcher {
 
     // group by extension
     for file in files {
-      let fileInfo = FileInfo(file)
-      if let mediaType = Utility.mediaType(forExtension: fileInfo.ext) {
-        filesGroupedByMediaType[mediaType]!.append(fileInfo)
+      autoreleasepool{
```

Comment:
If the performance thing about character stands, it means there will be a lot of `FileInfo`. So...why not `autorelease` it?